<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>load_data_form</class>
 <widget class="QDialog" name="load_data_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Geolimi - Chargement de données</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="toolTip">
      <string>Charger les données de la base dans QGIS</string>
     </property>
     <property name="title">
      <string>Styles à appliquer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="rightMargin">
       <number>25</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Thème</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbTheme">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Charger les données de la base dans QGIS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCharger">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Charger les données de la base dans QGIS</string>
     </property>
     <property name="text">
      <string>Charger les données</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pbProcess">
     <property name="toolTip">
      <string>Charger les données de la base dans QGIS</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bboxFermer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
